web/src/engine/websites/Imgur.ts

```diff
@@ -0,0 +1,45 @@
+import { Tags } from '../Tags';
+import icon from './Imgur.webp';
+import { DecoratableMangaScraper, Manga, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchWindowScript } from '../platform/FetchProvider';
+
+const pageScript = `
```

Comment:
Use `ScriptInjection` type
```typescript
function pageSript(): Promise<...> {
    return new Promise(resolve => {
        // ...
    });
}

Reply:
Common.PagesSinglePageJS will only accept a string.

I can do that for mangaScript tho.